submissions/dashakim/memory-game/script.js

```diff
@@ -1,90 +1,96 @@
-const section = document.querySelector("section");
+const section = document.querySelector("section.memory-game");
 
-const imagesNames = function(currentImg) {
+const getImagesNamesForAltAttribute = function(currentImg) {
   return currentImg
     .split(".")
     .slice(0, 1)
     .join("")
     .substr(4);
 };
 
-const createFrontFaceCard = function(div) {
+const createFrontFaceCard = function(cardsContainer) {
   const frontImage = document.createElement("img");
   frontImage.classList.add("front-face");
   frontImage.setAttribute("src", frontImageQuestion);
   frontImage.setAttribute("alt", "icon-question");
-  div.appendChild(frontImage);
+  cardsContainer.appendChild(frontImage);
 };
 
-const createBackFaceCard = function(div, el) {
+const createBackFaceCard = function(cardsContainer, currentImg) {
   const backImage = document.createElement("img");
   backImage.classList.add("back-face");
-  backImage.setAttribute("src", el);
-  backImage.setAttribute("alt", imagesNames(el));
-  div.appendChild(backImage);
+  backImage.setAttribute("src", currentImg);
+  backImage.setAttribute("alt", getImagesNamesForAltAttribute(currentImg));
+  cardsContainer.appendChild(backImage);
 };
 
-const createCard = function(storage) {
-  storage.map(i => {
+const createCards = function(storage) {
+  storage.map(e => {
     const divMemoryCard = document.createElement("div");
     divMemoryCard.classList.add("memory-card");
     section.appendChild(divMemoryCard);
-    divMemoryCard.dataset.framework = imagesNames(i);
+    divMemoryCard.dataset.framework = getImagesNamesForAltAttribute(e);
     createFrontFaceCard(divMemoryCard);
-    createBackFaceCard(divMemoryCard, i);
+    createBackFaceCard(divMemoryCard, e);
   });
 };
 
-createCard(images);
+createCards(images);
 
-const card = document.querySelectorAll(".memory-card");
+const cards = document.querySelectorAll(".memory-card");
 let lockBoard = false;
 let hasFlippedCard = false;
 let firstCard, secondCard;
 
-function flipCard() {
+const flipCard = e => {
   if (lockBoard) return;
-  if (this === firstCard) return;
```

Comment:
Since you rid of this check - some things are broken now. Try to play the game and detect what is going wrong.